code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -70,6 +70,7 @@
 	var/modtype = "Default"
 	var/lower_mod = 0
 	var/jetpack = 0

```

Comment:
If this is a boolean, and all code using it does't do much beyond `if (jetpack)` or what have you, then this can be edited to simply be `var/obj/item/weapon/tank/jetpack/carbondioxide/synthetic/jetpack = null` and you can get rid of `jetpackRef`.

An object variable having an ingame referenced assigned returns the same way a boolean set to true does. And if it's null, with no reference, it'll return the same way a boolean set to negative does. 99% of the time, this can be used.